plugins/ms-vscode.node-debug2/1.31.1/meta.yaml

```diff
@@ -0,0 +1,10 @@
+id: ms-vscode.node-debug2
```

Comment:
Can you elaborate why we need 2 instances of this plugin?

Reply:
They are two different node debug plugins
* ms-vscode.node-debug2 for node version >= 8.0 
* ms-vscode.node-debug2 for node version < 8.0

Reply:
Could you make titles differ?

Reply:
The titles are different already. What would you propose?

Reply:
I believe they are still the same: Node.js debugging support